.goreleaser.yaml

```diff
@@ -32,9 +32,24 @@ changelog:
   use: github
 
 dockers:
+  - build_flag_templates:
```

Comment:
The [docs](https://goreleaser.com/cookbooks/multi-platform-docker-images/) also mention it's important to include `use: buildx`

Reply:
That doc is much more useful than the [one](https://goreleaser.com/customization/docker_manifest/) I was reading / copying from.